pallets/stateful-storage/Cargo.toml

```diff
@@ -30,29 +30,14 @@ sp-runtime = { git = "https://github.com/paritytech/substrate", default-features
 sp-std = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.36" }
 # Frequency related dependencies
 common-primitives = { default-features = false, path = "../../common/primitives" }
+twox-hash = { version = "1.6.3", default-features = false, features = ["digest_0_10"] }
```

Comment:
nit: This should be above `# Frequency related dependencies` since it's not a frequency crate

Reply:
Fixed.